ako-operator/config/crd/bases/ako.vmware.com_akoconfigs.yaml

```diff
@@ -270,6 +275,9 @@ spec:
                   pspEnable:
                     type: boolean
                 type: object
+              replicaCount:
```

Comment:
why are we adding this again? I think this was part of release-1.9.1-operator.

Reply:
Actually, release-1.9.3 was branched out from master before release-1.9.1-operator changes could be cherry-picked back to master. Hence, release-1.9.1-operator changes are not present in release-1.9.3-operator(which was branched out from release-1.9.3).

Reply:
Please create a cherry-pick to master once this PR is merged.